Sources/KlaviyoCore/AppContextInfo.swift

```diff
@@ -30,61 +37,82 @@ public struct AppContextInfo {
         return deviceModel
     }()
 
-    private static let deviceIdStoreKey = "_klaviyo_device_id"
+    let defaultKlaviyoSdk = {
+        let plist = loadPlist(named: "klaviyo-sdk-configuration") ?? [:]
```

Comment:
Is there supposed to be a plist with this name?

Reply:
Yah in react-native and flutter. It's a way to override the sdk name.

Reply:
definitely out of scope of this pr, but just want to note that this only works on react native and flutter if the swift package is imported as a static dependency